cisco_secure_web_appliance/assets/logs/cisco-secure-web-appliance.yaml

```diff
@@ -0,0 +1,438 @@
+    path: network.client.geoip.subdivision.name
+    source: log
+  - groups:
+      - Web Access
+    name: Client IP
+    path: network.client.ip
+    source: log
+  - groups:
+      - Web Access
+    name: Client Port
+    path: network.client.port
+    source: log
+  - groups:
+      - Geoip
+    name: Destination City Name
+    path: network.destination.geoip.city.name
+    source: log
+  - groups:
+      - Geoip
+    name: Destination Continent Code
+    path: network.destination.geoip.continent.code
+    source: log
+  - groups:
+      - Geoip
+    name: Destination Continent Name
+    path: network.destination.geoip.continent.name
+    source: log
+  - groups:
+      - Geoip
+    name: Destination Country ISO Code
+    path: network.destination.geoip.country.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Destination Country Name
+    path: network.destination.geoip.country.name
+    source: log
+  - groups:
+      - Geoip
+    name: Destination Subdivision ISO Code
+    path: network.destination.geoip.subdivision.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Destination Subdivision Name
+    path: network.destination.geoip.subdivision.name
+    source: log
+  - groups:
+      - Web Access
+    name: Destination IP
+    path: network.destination.ip
+    source: log
+  - groups:
+      - Web Access
+    name: Destination Port
+    path: network.destination.port
+    source: log
+  - groups:
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Cisco Secure Web Appliance
+  enabled: true
+  filter:
+    query: "source:cisco-secure-web-appliance"
+  processors:
+    - type: pipeline
+      name: Processing of Access Logs
+      enabled: true
+      filter:
+        query: "service:access_logs "
+      processors:
+        - type: grok-parser
+          name: "Parse access logs"
+          enabled: true
+          source: message
+          samples:
+            - '<38>Sep 23 11:14:10 Crest.CSWA accesslogs: Info: 1727070241.436
+              866 10.10.10.10 TCP_MISS/200 4394 CONNECT
+              tunnel://v10.events.data.microsoft.com:443/ -
+              DIRECT/v10.events.data.microsoft.com application/octet-stream
+              DEFAULT_CASE_12-DefaultGroup-match_network-NONE-NONE-NONE-DefaultGroup-NONE
+              <"IW_comp",9.0,1,"-",-,-,-,1,"-",-,-,-,"-",1,-,"-","-",-,-,"IW_comp",-,"-","Computers
+              and Internet","-","Microsoft Dynamics CRM","Enterprise
+              Applications","Encrypted","-",40.59,0,-,"-","-",1,"-",-,-,"-","-",-,1,"-",-,->
+              - -'
+            - 1726663202.810 4 10.10.10.10 TCP_DENIED/403 0 CONNECT
+              tunnel://FORCEPOINTDLP:443/ - NONE/- -
+              BLOCK_WEBCAT_12-DefaultGroup-match_network-NONE-NONE-NONE-NONE-NONE
+              <"nc",ns,1,"-",-,-,-,-,"-",-,-,-,"-",-,-,"-","-",-,-,"nc",-,"-","-","-","Unknown","Unknown","-","-",0.00,0,-,"-","-",-,"-",-,-,"-","-",-,-,"-",-,->
```

Comment:
Why are some of the attributes in this list `-` but others are `"-"`?

Reply:
Referring to the official documentation of [cisco secure web appliance](https://www.cisco.com/c/en/us/td/docs/security/wsa/wsa-14-5/user-guide/wsa-userguide-14-5/b_WSA_UserGuide_11_7_chapter_010101.html#con_1633384), we have included this sample. Please note that the grok parser is designed to handle attributes seamlessly, regardless of whether they are enclosed in double quotes or not.